tcn_hpl/callbacks/plot_metrics.py

```diff
@@ -248,54 +266,56 @@ def on_validation_epoch_end(
 
         current_epoch = pl_module.current_epoch
         curr_acc = pl_module.val_acc.compute()
-        best_acc = pl_module.val_acc_best.compute()
-
-        class_ids = np.arange(all_probs.shape[-1])
-        num_classes = len(class_ids)
-
-        #
-        # Plot per-video class predictions vs. GT across progressive frames in
-        # that video.
-        #
-        # Build up mapping of truth to preds for each video
-        per_video_frame_gt_preds = defaultdict(dict)
-        for (gt, pred, prob, source_vid, source_frame) in zip(
-            all_targets, all_preds, all_probs, all_source_vids, all_source_frames
-        ):
-            per_video_frame_gt_preds[source_vid][source_frame] = (int(gt), int(pred))
-
-        plot_gt_vs_preds(self.output_dir, per_video_frame_gt_preds, split="validation")
+        curr_f1 = pl_module.val_f1.compute()
+        best_f1 = pl_module.val_f1_best.compute()
 
         #
         # Create confusion matrix
         #
+        class_ids = np.arange(all_probs.shape[-1])
```

Comment:
Is this a repeat of line 199?

Reply:
Yes, but these are different methods that are doing this with different inputs. I'm going to blame github of not showing you enough context.

Reply:
`def on_train_epoch_end` vs. `def on_validation_epoch_end` vs. `def on_test_epoch_end`